src/Microsoft.OData.Core/ODataMessageReaderSettings.cs

```diff
@@ -235,6 +236,12 @@ public ODataMessageQuotas MessageQuotas
         /// </summary>
         internal bool ThrowOnUndeclaredPropertyForNonOpenType { get; private set; }
 
+        /// <summary>
+        /// Require Content-Id header in changesets
+        /// If turned off allows to read OData 2.0 requests without Content-Id header present.

```

Comment:
```suggestion
        /// When set to false, it allows to reading OData 2.0 requests without Content-Id header present. Otherwise, an exception is thrown.

```